build/ci.go

```diff
@@ -334,10 +344,14 @@ func doLint(cmdline []string) {
 
 // downloadLinter downloads and unpacks golangci-lint.
 func downloadLinter(cachedir string) string {
-	const version = "1.39.0"
+	const version = "1.42.0"
```

Comment:
The latest version goes to `1.47.2` now.

Reply:
I will try to update this